guide/src/SUMMARY.md

```diff
@@ -22,6 +22,7 @@ limitations under the License.
 - [How to initialize a client](initialize_a_client.md)
 - [Configuring retry policies](configuring_retry_policies.md)
 - [Error handling](error_handling.md)
+- [Examine error details](examine_error_details.md)
```

Comment:
comment: we might want to organize these into sections, but let's defer that until we have more guides

Reply:
Ack